src/wrapped/wrappedlibv4l2_private.h

```diff
@@ -0,0 +1,10 @@
+#if !(defined(GO) && defined(GOM) && defined(GO2) && defined(DATA))
+#error Meh...
+#endif
+
+GO(v4l2_open, iFpi)
+GO(v4l2_close, iFi)
+GO(v4l2_ioctl, iFiii)
+GO(v4l2_read, iFipu)
+GO(v4l2_mmap, pFpuiiiI)
```

Comment:
The mmap and munmap may need need a "GOM()"  wrapping (so writing a "my_" function), to allow box86 to track the memory mapped and unmapped (like with libc `mmap(...)` and `munmap(...)`, but without the executable flag tracking...

Reply:
got it, I will modify and make pull request again